File: Desafio - JAVASCRIPT/Verificar_Url.js
function verificarUrl(url) {
    if (url.startsWith("https://")) {
        return "URL segura";
    } else if (url.startsWith("http://")) {
        return "URL não segura";
    } else {
        return "Formato inválido";
    }
}

// Entrada do usuário
const url = prompt("Digite a URL do site em questão:");
const resultado = verificarUrl(url);

// Exibe o resultado
console.log(resultado);

File: Desafio - PYTHON/Verificar_Url.py
def verificar_url(url):
    if url.startswith("https://"):
        return "URL segura"
    elif url.startswith("http://"):
        return "URL não segura"
    else:
        return "Formato inválido"

# Entrada do usuário
url = input("Digite a URL do site em questão: ")
resultado = verificar_url(url)

# Exibe o resultado
print(resultado)
